test/stubs/talon/__init__.py

```diff
@@ -184,6 +184,9 @@ class Resource:
     def open(self, path: str, mode: str = "r"):
         return open(path, mode, encoding="utf-8")
 
+    def watch(self, path: str):
+        pass
```

Comment:
Shouldn't this return an identity function?

```suggestion
        return lambda f: f
```